src/app/docs/navigation/demo-tabs/demo-tabs.component.html

```diff
@@ -1,146 +1,151 @@
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                    <li>
-                        <div>Star wars</div>
-                        <div>
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                    <li>
-                        <div>Pulp fiction</div>
-                        <div>
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                    <li>
-                        <div>Watchmen</div>
-                        <div>
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                    <li>
-                        <div>Joker</div>
-                        <div>
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                    <li>
-                        <div>Gladiator</div>
-                        <div>
-                            <sdk-checkbox></sdk-checkbox>
-                        </div>
-                    </li>
-                </ul>
-            </div>
-        </sdk-tab>
-        <sdk-tab label="Tab 3">
-            <div>
-                <h1>Title 3</h1>
-                Content 3
-            </div>
-        </sdk-tab>
-    </sdk-tab-group>
-</div>
+<lib-demo-page [page]="page">
+    <ng-template #basicRef>
+        <div>
+            <h1>Tabs</h1>
+            <sdk-tab-group [animation]="true">
+                <sdk-tab label="Tab 1" icon="star">
+                    <div class="tab-wrapper">
+                        <h2 class="demo-tabs__title">Pick color</h2>
+                        <sdk-radio [data]="[{value: 0, name: 'red'}, {value: 1, name: 'blue'}, {value: 2, name: 'green'}]">
+                        </sdk-radio>
+                        <h2 class="demo-tabs__title">Pick fruit</h2>
+                        <sdk-radio
+                            [data]="[{value: 0, name: 'mango'}, {value: 1, name: 'banana'}, {value: 2, name: 'mandarin'}]"></sdk-radio>
+                        <h2 class="demo-tabs__title">Pick drink</h2>
+                        <sdk-radio
+                            [data]="[{value: 0, name: 'milkshake'}, {value: 1, name: 'kombucha'}, {value: 2, name: 'smoothie'}]"></sdk-radio>
+                    </div>
+                </sdk-tab>
+                <sdk-tab label="Disabled Tab" [disabled]="true"></sdk-tab>
+                <sdk-tab label="Super-mega-uber-extra-duper-long-tab">
+                    <div class="tab-wrapper">
+                        <h2 class="demo-tabs__title">Pick color</h2>
+                        <sdk-radio
+                            [data]="[{value: 0, name: 'red'}, {value: 1, name: 'blue'}, {value: 2, name: 'green'}]"></sdk-radio>
+                        <h2 class="demo-tabs__title">Pick fruit</h2>
```

Comment:
use `*ngFor="let x of arr"` for example templates, not a hardcoded ones
f.e.
```html
<sdk-tab-group [animation]="true">
  <sdk-tab *ngFor="let t of tabs" label="t.label" [icon]="t.icon" [disabled]="t.disabled">
      <!-- content injection via ngTemplate/Ref -->
  </sdk-tab>
</sdk-tab-group>
```

Reply:
done